### PR TITLE
NH-76996 - Disabled platform events log processing

### PR DESCRIPTION
**Pull request by jerrytfleung**
Disabled platform events log processing

https://swicloud.atlassian.net/browse/NH-76996

### CODE REVIEW

File: collector/receiver/telemetryapireceiver/receiver.go [hunk 1]
```diff
@@ -216,16 +216,16 @@ func (r *telemetryAPIReceiver) createLogs(slice []telemetryapi.Event) (plog.Logs
...
-		logRecord := scopeLog.LogRecords().AppendEmpty()
-		logRecord.Attributes().PutStr("type", el.Type)
-		if t, err := time.Parse(timeFormatLayout, el.Time); err == nil {
-			logRecord.SetTimestamp(pcommon.NewTimestampFromTime(t))
-			logRecord.SetObservedTimestamp(pcommon.NewTimestampFromTime(time.Now()))
-		} else {
-			r.logger.Error("error parsing time", zap.Error(err))
-			return plog.Logs{}, err
-		}
 		if el.Type == string(telemetryapi.Function) || el.Type == string(telemetryapi.Extension) {
```
Comment (cheempz) [inline collector/receiver/telemetryapireceiver/receiver.go:219]: I thought we agreed not to forward Extension events as logs?

Comment (jerrytfleung) [reply collector/receiver/telemetryapireceiver/receiver.go:219]: Yes, the logic is the same. The decision now is made from the subscription type. 

https://github.com/solarwinds/opentelemetry-lambda/blob/no_platform/collector/receiver/telemetryapireceiver/receiver.go#L74

Comment (cheempz) [reply collector/receiver/telemetryapireceiver/receiver.go:219]: I see. Can we just not subscribe to anything other than Function events for this round to minimize overhead?

Comment (jerrytfleung) [reply collector/receiver/telemetryapireceiver/receiver.go:219]: Sure. Done

**Comment by jerrytfleung** [comment]
Disabled log forwarding by commenting out logs pipeline in config.yaml

**Review by cheempz — APPROVED** [review]
Great, thanks!

**Pull request merged**